apps/journeys-admin/src/components/Editor/Slider/Content/Goals/GoalsList/GoalsList.spec.tsx

```diff
@@ -19,7 +18,7 @@ import { GoalsList } from './GoalsList'
 
 jest.mock('@mui/material/useMediaQuery', () => ({
   __esModule: true,
-  default: jest.fn()
+  default: jest.fn(() => false)
```

Comment:
```suggestion
  default: jest.fn()
```
Do we need to return false here? the tests pass whether its empty or false